src/coreml.cs

```diff
@@ -507,6 +524,11 @@ interface MLModelMetadata {
 	[BaseType (typeof (NSObject))]
 	interface MLMultiArray : NSSecureCoding {
 
+		[Deprecated (PlatformName.MacOSX, 13, 0, message: "Use 'GetBytesWithHandler' or 'GetMutableBytes' async methods instead.")]
+		[Deprecated (PlatformName.iOS, 16, 0, message: "Use 'GetBytesWithHandler' or 'GetMutableBytes' async methods instead.")]
+		[Deprecated (PlatformName.TvOS, 16, 0, message: "Use 'GetBytesWithHandler' or 'GetMutableBytes' async methods instead.")]
+		[Deprecated (PlatformName.WatchOS, 9, 0, message: "Use 'GetBytesWithHandler' or 'GetMutableBytes' async methods instead.")]
+		[Deprecated (PlatformName.MacCatalyst, 16, 0, message: "Use the asynchronous interface instead.")]
```

Comment:
Why is this message different to the ones above?

Reply:
Cause I yy and missed it